### PR TITLE
Use publish_date as fallback if publish_date_override is not set

### PR DESCRIPTION
**Pull request by lhh**
Signed-off-by: Lon Hohberger <lhh@redhat.com>

### CODE REVIEW

**Comment by yazug** [comment]
looks good to me

File: errata_tool/erratum.py [hunk 1]
```diff
@@ -1011,6 +1018,9 @@ def dump(self):
         if self.publish_date_override is not None:
             print('')
             print("Ship Target: {0}".format(self.publish_date_override))
+        elif self.publish_date is not None:
+            print('')
+            print("Ship Target: {0}".format(self.publish_date))
```
Comment (ktdreyer) [inline errata_tool/erratum.py]: We're printing two identical "Ship Target" lines here, which seems confusing. How can we distinguish this line from the `print("Ship Target:" ... )` above?

File: errata_tool/erratum.py [hunk 2]
```diff
@@ -45,6 +45,7 @@ def _do_init(self):
         self.text_only = False
         self.text_only_cpe = None
         self.publish_date_override = None
+        self.publish_date = None
```
Comment (ktdreyer) [inline errata_tool/erratum.py]: Let's add a unit test to cover this new attribute. In `errata_tool/tests/test_advisory.py`, you can add a new method like this:

```python
    def test_publish_date(self, advisory):
        assert advisory.publish_date == '2018-Sep-26'
```

Comment (lhh) [reply errata_tool/erratum.py]: Done

File: errata_tool/erratum.py (hunk 1 above)
Comment (ktdreyer) [reply errata_tool/erratum.py]: From @lhh on IRC, "the `publish_date` is the default one set - perhaps by setting an errata group, or SRT setting an embargo date; the `publish_date_override` is set by the errata creator (devel can edit this one, but not the other)"

Comment (lhh) [reply errata_tool/erratum.py]: Ah, I missed this part. I'll update.

**Pull request merged**